internal/common/retry/interceptor.go

```diff
@@ -117,8 +117,7 @@ var (
 	// gRPC response codes that represent retryable errors.
 	// The following status codes are never generated by the library:
 	//    INVALID_ARGUMENT, NOT_FOUND, ALREADY_EXISTS, FAILED_PRECONDITION, ABORTED, OUT_OF_RANGE, DATA_LOSS
-	retryableCodes = []codes.Code{codes.Aborted, codes.DeadlineExceeded, codes.Internal,
-		codes.ResourceExhausted, codes.Unavailable, codes.Unknown}
+	retryableCodes = []codes.Code{codes.Aborted, codes.DeadlineExceeded, codes.Internal, codes.Unavailable, codes.Unknown}
```

Comment:
@alexshtin can you confirm that we are not using ResourceExhausted error for server side throttling?  If that is the case than it looks good.

Reply:
This error is particularly messy. We do use it for throttling and for some non-retryable errors like `ErrNamespaceCountLimitServerBusy`, `ErrSizeExceedsLimit`.
My thinking was that in case of QPS errors, we probably don't want to bombard server with retries (since our default policy is pretty aggressive 20ms with 1.2x backoff), because in this case we would probably be getting the same error again. I'm not sure if we should keep this one retryable or not, let's have a discussion.
I think it's probably better to leave it as it is, the tradeoff is that we won't see a clear message in case if message size exceeds limits, but default limit is very high so nobody should be hitting it anyways. On the other hand removing retry may result in potential issues.